alacritty/src/display.rs

```diff
@@ -471,7 +478,17 @@ impl Display {
 
             self.renderer.with_api(&config.ui_config, config.cursor, &size_info, |mut api| {
                 // Iterate over all non-empty cells in the grid.
-                for cell in grid_cells {
+                for mut cell in grid_cells {
+                    // Invert the active match in vi-less search.
+                    if search_active && viewport_match.contains(&Point::new(cell.line, cell.column))
+                    {
+                        let colors = config.colors.search.focused_match;
+                        let match_fg = colors.foreground().color(cell.fg, cell.bg);
+                        cell.bg = colors.background().color(cell.fg, cell.bg);
+                        cell.fg = match_fg;
+                        cell.bg_alpha = 1.0;
+                    }
```

Comment:
Since we're applying that logic after selection is done you hide selection with active match, which I'm not really a fan, however doing it the right way will mean doing it in renderable cells iter, which is becoming messy, however I'm not sure how to resolve this issue besides applying active match color before selection.